pkg/stub/secrets.go

```diff
@@ -15,6 +15,7 @@ import (
 const (
 	coreosPullSecretNamespace = "openshift-config"
 	coreosPullSecretName      = "pull-secret"
+	metricsSecret             = "samples-operator-tls"
```

Comment:
is this being used somewhere?

Reply:
yeah in the end only the deployment needed references to that after I switched to the library-go watch approach.

I'll remove.